README.md

```diff
@@ -110,10 +110,15 @@ The result will be named as `<name>-<version>.<type>`.
 By default, the application name is detected from the rockspec, and
 the application version is detected from `git describe`.
 
-#### General packing flow and options
+#### Build directory
+
+By default, application build is performed in the temporarily directory, so the packaging process doesn't affect the contents of your application directory.
+
+You can specify custom build directory for your project in `TARANTOOL_BUILDDIR` environment variable. If this directory doesn't exists, it will be created, used for building the application and then removed.
 
-A package is first created in a temporarily directory, so the packaging process
-doesn't affect the contents of your application directory.
+If you specify existent directory in `TARANTOOL_BUILDDIR` environment variable, `TARANTOOL_BUILDDIR/cartridge.build` repository will be used for build and then removed. If `TARANTOOL_BUILDDIR/cartridge.build` is already exists, it will be cleaned before building application.
```

Comment:
Please, split long paragraphs into several lines. Markdown supports it.

It could be quite hard to edit in future in some editors